### PR TITLE
MIG-683: Debug resources and updated log sections

### PR DESCRIPTION
**Pull request by apinnick**
https://issues.redhat.com/browse/MIG-683

Add Migration resources tree to Troubleshooting and update log modules.

4.5+

Previews:

https://deploy-preview-33588--osdocs.netlify.app/openshift-enterprise/latest/migrating_from_ocp_3_to_4/troubleshooting-3-4.html#migration-viewing-migration-plan-log_troubleshooting-3-4

https://deploy-preview-33588--osdocs.netlify.app/openshift-enterprise/latest/migrating_from_ocp_3_to_4/troubleshooting-3-4.html#migration-viewing-migration-plan-resources_troubleshooting-3-4

QE approved

### CODE REVIEW

**Comment by apinnick** [comment]
@XinRedhat  
Ready for QE review.

File: modules/migration-viewing-migration-plan-resources.adoc [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+You can view the resources of a migration plan to monitor the progress of a live migration or to troubleshoot a failed migration. You use the {mtc-short} web console to copy a command to your clipboard. Then, you run the command from the command line interface (CLI).
+
+If you are troubleshooting a failed migration, start with a high-level resource that has failed and then work down the resource tree, towards the lower-level resources.
+
+.Procedure
+
+. In the {mtc-short} web console, click *Migration Plans*.
+. Click the *Migrations* number next to a migration plan to view the *Migrations* page.
++
+The *Migrations* page displays the migration types associated with the migration plan, for example, *Stage* or *Cutover* for warm migration.
```
Comment (XinRedhat) [inline modules/migration-viewing-migration-plan-resources.adoc:18]: @apinnick I am confused about  *Cutover*

File: modules/migration-viewing-migration-plan-resources.adoc [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+. Click the *Type* to view the *Migration details* page.
++
+The *Migration details* page displays the steps of the migration and their status.
+
+. Expand *Migration resources* to view the migration resources and their status.
+. Click the {kebab} next to a resource and select one of the following options:
+
+* *Copy `oc describe` command* copies the command to your clipboard. Log in to the relevant cluster and run the command from the CLI. The command displays the conditions and events of the resource as a YAML file.
+* *Copy `oc logs` command*: copies the command to your clipboard. Log in to the relevant cluster and run the command from the CLI. The command displays the logs of the resource. If the resource supports log filtering, the filtered log is displayed.
+* *View JSON*: The conditions and events of the resource are displayed as a JSON file in a web browser.
```
Comment (XinRedhat) [inline modules/migration-viewing-migration-plan-resources.adoc:29]: JSON content shows up the resource with json formal, it's similar to what `oc get` command. I think this description is not accurate

File: modules/migration-viewing-migration-plan-resources.adoc (hunk 1 above)
Comment (apinnick) [reply modules/migration-viewing-migration-plan-resources.adoc:18]: "Cutover" is the second "type" of warm migration. The Type field will show "stage" or "cutover" for warm migration, if they have been run.

Comment (XinRedhat) [reply modules/migration-viewing-migration-plan-resources.adoc:18]: @apinnick for MTC, it just has 3 types of migration, "stage", "migration", "rollback", I never heard the term "warm migration" on MTC.

Comment (apinnick) [reply modules/migration-viewing-migration-plan-resources.adoc:18]: You're correct. Thanks for catching that.

**Comment by XinRedhat** [comment]
LGTM @apinnick

**Pull request merged**